### DISCUSSION TITLE
Manual instrumentation name

### DISCUSSION THREAD

**Discussion by APIM-KiranVaddadi** [open-telemetry/opentelemetry-java-instrumentation#8217] [Q&A] [answered]
I am trying to instrument a java application. How can I set the instrumentation name.
Using io.opentelemetry.api.GlobalOpenTelemetry to get Meter and Tracer. Not sure where to set the instrumentation name.

Thanks,
Kiran

**Comment by mateuszrzeszutek** [comment]
Hey @APIM-KiranVaddadi ,
You can use the name of your application, for instance. The purpose of the instrumentation name is to identify the source of telemetry, you can pick pretty much any identifier you want. For example:
```java
GlobalOpenTelemetry.getTracer("my-app")
```

**Reply by APIM-KiranVaddadi** [reply]
Thank you for your response. 
Isn't the Instrumentation Scope Name that is passed in the above method? 
https://github.com/open-telemetry/opentelemetry-java/blob/main/api/all/src/main/java/io/opentelemetry/api/GlobalOpenTelemetry.java#L129

Is 'instrumentation scope' and 'instrumentation name' same?
I would like to enable disable traces using env properties like below. It did not work using above approach.
OTEL_INSTRUMENTATION_[instrumentation_name]_ENABLED=false

**Reply by trask** [reply, marked as answer]
> OTEL_INSTRUMENTATION_[instrumentation_name]_ENABLED=false

this configuration is only supported by javaagent instrumentation

there isn't a more general solution today, you may be interested in following https://github.com/open-telemetry/opentelemetry-specification/issues/1588

**Reply by APIM-KiranVaddadi** [reply]
Thank you.
